tools/generate-helpers/pg-table-bindings/testgraphtables/testchild1/store_tx_test.go

```diff
@@ -0,0 +1,98 @@
+//go:build sql_integration
+
+package postgres
```

Comment:
This is piggy back the existing test suite I can add this to all the stores eventually.